src/Nav/Nav.js

```diff
@@ -14,12 +14,14 @@ const Nav = (props) => {
       </button> 
       <button 
         className={props.isLoggedIn ? 'hidden' : ''}
-        id='login-btn'>
+        id='login-btn'
+        onClick={props.pageView}>
```

Comment:
Something got mixed up here and the `showLoginPage` function isn't getting passed. I've got a fixed copy already, let me know if you'd like me to push it to the branch before we merge.

Reply:
Yes go for it!